assignments/objects.js

```diff
@@ -19,46 +19,116 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi = {
+  id: 1,
+  name: 'Mitzi',
+  email: 'mmelloy@psu.edu',
+  gender: 'F'
+}
+
+const Kennan = {
+  id: 2,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com',
+  gender: 'M',
+  speak: function speak() {
+    console.log(`Hello, my name is Kennan!`)
+  }
+}
+
+const Keven = {
+  id: 3,
+  name: 'Keven',
+  email: 'kmummery2@wikimedia.org',
+  gender: 'M'
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+const Gannie = {
+  id: 4,
+  name: 'Gannie',
+  email: 'gmartinson3@illinois.edu',
+  gender: 'M'
+}
+
+const Antonietta = {
+  id: 5,
+  name: 'Antonietta',
+  email: 'adaine5@samsung.com',
+  gender: 'F',
+  multiplyNums: function multiplyNums(x, y) {
+    console.log(x*y);
+  }
+}
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(Mitzi['name']);
 // Kennan's ID
-
+console.log(Kennan['id']);
 // Keven's email
-
+console.log(Keven['email']);
 // Gannie's name
-
+console.log(Gannie['name']);
 // Antonietta's Gender
+console.log(Antonietta['gender']);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+Kennan.speak();
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+Antonietta.multiplyNums(3,4);
+
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
+// ==== Stretch Challenge: Nested Objects and the this keyword ====
 
 // 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
+  age: 70,
+  speak: function speak() {
+    console.log(`Hi, my name is ${'name'}!`);
```

Comment:
You had the right idea, but this only returns a 'name' string. First, you don't need quotes inside a template literal, you can just call your JS variables directly. Second, you needed some more info to reach the specific name properties you were looking for.

```suggestion
    console.log(`Hi, my name is ${parent.name}!`);
```